### PR TITLE
Add #6102 as a feature highlight

### PR DESCRIPTION
**Pull request by psobolewskiPhD**
I've added a description of https://github.com/napari/napari/pull/6102 to the top of the highlights as it's the only real feature added to 0.4.19. Also I made a note of the change in behavior with `orange`.
Hopefully it's clear -- @jni can you take a look?

### CODE REVIEW

File: highlight/0.4.19.md [hunk 1]
```diff
@@ -2,6 +2,8 @@ This release is mostly a bug fix release. We aim for the next release to be 0.5.
 
 Highlights in `v0.4.19`:
 
+A new feature: [napari/napari/#6102](https://github.com/napari/napari/pull/6102) added convenient parsing of color names and hex color strings in layer data and generating black-to-color colormaps when adding layers to the viewer. For example, one can now use: `viewer.add_image(data, colormap="crimson")` or `viewer.add_image(data, colormap="#FAFA00")`. To see what color names can be handled, please refer to the [`vispy` dictionary of colors](https://github.com/vispy/vispy/blob/269ed1ac4d8126421fd5a7eb06a2996d63f46b17/vispy/color/_color_dict.py#L181). Note: there is a namespace colision for the case of `orange` which is both a proper `vispy` colormap name and a color name. Previosuly, `orange` was handled as the `vispy` colormap, which is from white to orange. However, with 0.4.19, `orange` will be handled as a color and the generated colormap will be from black to orange.
```
Comment (melissawm) [inline highlight/0.4.19.md:5]:
```suggestion
A new feature: [napari/napari/#6102](https://github.com/napari/napari/pull/6102) added convenient parsing of color names and hex color strings in layer data and generating black-to-color colormaps when adding layers to the viewer. For example, one can now use: `viewer.add_image(data, colormap="crimson")` or `viewer.add_image(data, colormap="#FAFA00")`. To see what color names can be handled, please refer to the [`vispy` dictionary of colors](https://github.com/vispy/vispy/blob/269ed1ac4d8126421fd5a7eb06a2996d63f46b17/vispy/color/_color_dict.py#L181). Note: there is a namespace collision for the case of `orange` which is both a proper `vispy` colormap name and a color name. Previously, `orange` was handled as the `vispy` colormap, which is from white to orange. However, with 0.4.19, `orange` will be handled as a color and the generated colormap will be from black to orange. 
```

**Review by melissawm — COMMENTED** [review]
Reads clear to me, and I'm totally a layperson here.

File: highlight/0.4.19.md [hunk 2]
```diff
@@ -2,6 +2,8 @@ This release is mostly a bug fix release. We aim for the next release to be 0.5.
 
 Highlights in `v0.4.19`:
 
+A new feature: [napari/napari/#6102](https://github.com/napari/napari/pull/6102) added convenient parsing of color names and hex color strings in layer data and generating black-to-color colormaps when adding layers to the viewer. For example, one can now use: `viewer.add_image(data, colormap="crimson")` or `viewer.add_image(data, colormap="#FAFA00")`. To see what color names can be handled, please refer to the [`vispy` dictionary of colors](https://github.com/vispy/vispy/blob/269ed1ac4d8126421fd5a7eb06a2996d63f46b17/vispy/color/_color_dict.py#L181). Note: there is a namespace collision for the case of `orange` which is both a proper `vispy` colormap name and a color name. Previously, `orange` was handled as the `vispy` colormap, which is from white to orange. However, with 0.4.19, `orange` will be handled as a color and the generated colormap will be from black to orange.
```
Comment (jni) [inline highlight/0.4.19.md:5]: I think that switching the default behaviour of "orange" is the right call, but it should be (also) added to the API Changes section.

@Czaki would updating the tags on napari/napari#6102 update the cherry picking to add the PR to the API changes section? Or is it fixed once you have cherry-picked it once?

btw @psobolewskiPhD note the syntax for is `org/repo#678`, not `org/repo/#678` (extra slash)

Comment (Czaki) [reply highlight/0.4.19.md:5]: > @Czaki would updating the tags on [napari/napari#6102](https://github.com/napari/napari/pull/6102) update the cherry picking to add the PR to the API changes section? Or is it fixed once you have cherry-picked it once?

My script checks tags every time I generate release notes, so update tags will reflect the next generated notes.  (And of course I will run this script before release). Should one add a `api` tag to this PR?

Comment (jni) [reply highlight/0.4.19.md:5]: Yes please do. It was unintentional but now we are intentionally keeping it! 😊

Comment (Czaki) [reply highlight/0.4.19.md:5]: Adding option to pass name of color or hex of color was intentional. And it is still api change. 

Or you also want to have breaking-change tag?

Comment (jni) [reply highlight/0.4.19.md]: Ah, yes, I thought API change meant breaking change. I don't think we need a tag for additions to the API.

**Pull request merged**